src/libs/actions/Report.js

```diff
@@ -659,9 +659,15 @@ function fetchOrCreateChatReport(participants, shouldNavigate = true) {
             Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT}${data.reportID}`, {reportID: data.reportID});
 
             if (shouldNavigate) {
-                // Redirect the logged in person to the new report
-                Navigation.navigate(ROUTES.getReportRoute(data.reportID));
+                try {
+                    // Redirect the logged in person to the new report
+                    Navigation.navigate(ROUTES.getReportRoute(data.reportID));
+                } catch (e) {
+                    // When switching between the Auth and Public routers there's a brief time of no navigator mounted
+                    console.error(e);
+                }
             }
```

Comment:
This is executed after the Navigation was initialized (when `PublicScreens` mounted) but before the `AuthScreens` are mounted and throws an error. When the error is not handled the promise chain is broken and the original caller never resolves which in turn prevents rendering the initial state of the app

Reply:
This would be reverted as well if we use a different approach